README.md

```diff
@@ -1,7 +1,7 @@
 # Atom IDE Scala language client
 
 [![](https://img.shields.io/github/release/laughedelic/atom-ide-scala/all.svg)](https://github.com/laughedelic/atom-ide-scala/releases/latest)
-[![](https://img.shields.io/badge/license-LGPLv3-blue.svg)](https://www.tldrlegal.com/l/lgpl-3.0)
+[![](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
Why suddenly yellow? 😄 I don't have anything against, but I've never seen a yellow license badge..
Also, could you decapitalize `License`? For consistency with other badges.